### ISSUE TITLE
SVN: Lack of needs update icon

### ISSUE THREAD

**Issue by dualsky** [pocorall/scm-notifier#15]
I use SCM Notifier version 11.11.23.

Sometimes the icon indicating that some working copy needs update disappears. I have never seen the same behavviour in the original SVN notifier with the same list of working copies, except "Java" Git repository of course.

I have both Git and SVN working copies defined in SCM repositories (as could be seen on following image):
![SCM Norifier - lack of needs update icon](https://cloud.githubusercontent.com/assets/7819201/6060746/be6ea842-ad40-11e4-8dc8-25509dfb80c6.png)

I cannot catch exact moment when the problem occurs, but I have a storng feeling, that this happens in following situation:
1. SCM Notifier works hidden in the tray.

2. Working copies relates to following Source URLs:
  * c:\projects\xxx\_production => http://svn.server.name.com/branches/current_production
  * c:\projects\xxx\_tests => http://svn.server.name.com/branches/current_test
  * c:\projects\xxx\free => http://svn.server.name.com/trunk
  * c:\projects\xxx\project => http://svn.server.name.com/trunk
  * c:\projects\xxx\project_cleen => http://svn.server.name.com/trunk

3. Probably change was noticed on trunk - you could see all trunk related working copies marked with yellow icon.

4. Then somebody merged the bugfix to current_test and current_productions branches.

**Comment by pocorall** [comment]
Yes, sometimes I see the icon is not displaying, but I also cannot figure out the exact reason.

**Comment by dualsky** [comment]
Hi,
I catch it again today, and I confirmed the reason I gave previously. 

I didn't update my repositories for whole Friday. Today (on Monday) I noticed that one icon is yellow and two are missing. I used *Right click* -> *Change Log...* and confirmed that the repository marked with yellow  icon was updated on Friday morning and these lacking icon were update in the afternoon.

**Comment by KohrAhr** [comment]
I suppose issue in code -- how it's reference to icons.
